contrib/win32/openssh/install-sshd.ps1

```diff
@@ -114,16 +120,21 @@ if (Test-Path $sshProgDataPath)
     }
 }
 
-#register etw provider
-wevtutil im `"$etwman`"
+# Register etw provider
+# PowerShell 7.3+ has new/different native command argument parsing
+if ($PSVersiontable.PSVersion -le '7.2.9') {
+    wevtutil im `"$etwman`"
+} else {
+    wevtutil im "$etwman"
+}
 
 $agentDesc = "Agent to hold private keys used for public key authentication."
-New-Service -Name ssh-agent -DisplayName "OpenSSH Authentication Agent" -BinaryPathName `"$sshagentpath`" -Description $agentDesc -StartupType Manual | Out-Null
+New-Service -Name ssh-agent -DisplayName "OpenSSH Authentication Agent" -BinaryPathName "$sshagentpath" -Description $agentDesc -StartupType Manual | Out-Null
```

Comment:
Will this be a problem on older versions of powershell? Should we have an if statement like the one in line 125?

Reply:
No, there is no ambiguity with quoting for PowerShell cmdlets, it is only for native commands.  The escaped quotes are just not needed.